packages/apollo-cli/src/commands/queries/extract.ts

```diff
@@ -0,0 +1,80 @@
+import 'apollo-codegen-core/lib/polyfills';
+import { Command, flags } from '@oclif/command';
+import * as Listr from 'listr';
+import { print } from 'graphql';
+import { sha512 } from 'js-sha512';
+import * as fs from 'fs';
+
+import { loadQueryDocuments } from 'apollo-codegen-core/lib/loading';
```

Comment:
Longer term, is it confusing to leave the `loadQueryDocuments` functionality in `apollo-codegen-core`? (It doesn’t really sound like a utility library and we’re not doing code-gen.)

Reply:
hmm we could probably make the judgement of apollo-codegen-core in general. Let's keep it in mind as we continue digging into the functionality